src/main/resources/db/migration/V1_49_1__contract_type_data.sql

```diff
@@ -0,0 +1,20 @@
+insert into contract_type(id, name)
```

Comment:
should we use this opportunity to remove the 'women's specific' service category from the existing table?